package.json

```diff
@@ -15,8 +15,8 @@
   "dependencies": {
     "@babel/compat-data": "^7.12.13",
     "@mdi/font": "^4.5.95",
-    "@sentry/browser": "^5.29.1",
-    "@sentry/integrations": "^5.29.1",
+    "@sentry/browser": "^6.2.1",
+    "@sentry/integrations": "^6.2.1",
```

Comment:
I checked the release notes and did a bit of smoke testing, and it looks like nothing is broken.